pkg/grpc/errors/interceptor.go

```diff
@@ -43,6 +43,13 @@ func PanicOnInvariantViolationStreamInterceptor(srv interface{}, ss grpc.ServerS
 func LogInternalErrorInterceptor(ctx context.Context, req interface{}, _ *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
 	resp, err := handler(ctx, req)
 	logErrorIfInternal(err)
+	if err != nil {
```

Comment:
How about inverting if? It will reduce indent

```suggestion
	if err == nil {
		return nil
	}
	message := errox.GetBaseSentinelMessage(err)
	if message != "" {
		return errors.New(message)
	}
	return errox.ServerError
```